src/coreclr/gc/gcinterface.h

```diff
@@ -1032,6 +1032,7 @@ struct ScanContext
 {
     Thread* thread_under_crawl;
     int thread_number;
+    int thread_count;
```

Comment:
this would break the backward compatibility with the VM side (ie, you can't run clrgc.dll with a previous version of coreclr.dll anymore). can you change it in a way that does not introduce this breaking change?